shepherd/DummyShepherd.py

```diff
@@ -0,0 +1,18 @@
+import queue
+import time
+from LCM import *
+from Utils import *
+
+if __name__ == '__main__':
+    while True:
+        msg = {"autonomous": True, "enabled": True}
+        lcm_send(LCM_TARGETS.DAWN, DAWN_HEADER.ROBOT_STATE, msg)
+
+        time.sleep(1)
+        msg2 = {"autonomous": False, "enabled": False}
+        lcm_send(LCM_TARGETS.DAWN, DAWN_HEADER.ROBOT_STATE, msg2)
+        time.sleep(1)
+
+        # lcm_send(LCM_TARGETS.DAWN, DAWN_HEADER.CODES, {"rfids" : [1, 2, 3, 4, 5, 6],
+        #                                                "codes" : [1, 2, 3, 4, 5, 6],
+        #                                                "solutions" : [1, 2, 3, 4, 5, 6]})
```

Comment:
Remove commented code. This is the point of version control. If you anticipate needing this alternate method call often, rearchitect your code so that if/else behavior can be controlled with a boolean switch.